07-ipcc.Rmd

```diff
@@ -1,11 +1,157 @@
+
+
+```{r radar_plot, echo=FALSE, out.width="100%", fig.cap="**Abbildung 1:** Radar plots illustrating the 24-hour distribution of heat stress risk across Southern, Western, Eastern, and Northern Europe during three decades: 1990–2000, 2001–2011, and 2012–2022 [@vandaalen2024]."}
```

Comment:
> ...three decades: 1990–2000, 2001–2011, and 2012–2022 [@vandaalen2024].

This won't work in the pdf rendering. Here is a minimal example of how to make cites in figure captions work:

(ref:captionScatter) @genest2013
```
```{r scatterClaudia, echo=FALSE, message=FALSE, warning=FALSE, out.width="60%", fig.align='center', fig.cap="Scatter plot of the pseudo-observations \\((u_1, v_1) \\ldots, (u_n, v_n)\\) from \\(C\\), where \\(u_i\\) is calculated from data on the Inn and \\(v_i\\) is obtained from data on the Danube (\\(\\forall i \\in \\{1,\\ldots, n\\}\\)). Adapted from (ref:captionScatter), p. 93."}
include_graphics("./work/04-archimax/figures/Plot_Inn_Danube_raw_adjusted_v02.png")
```
```